src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -318,8 +324,8 @@ export class MeetingManager implements AudioVideoObserver {
         await this.audioVideo?.chooseAudioInputDevice(
           this.audioInputDevices[0].deviceId
         );
-      } catch (e) {
-        console.error(`Error in selecting audio input device - ${e}`);
+      } catch (error) {
```

Comment:
Should we set and send the device selection error here as well?

Reply:
This is a good question. From my points, the issue is only about `selectXxxxxInputDeviceError()`, so I only change those functions. But generally, we may want to set and send the error wherever there is a calling to `audioVideo?.chooseXxxxxInputDevice()` API. How do you think? 
@ltrung could you give us some advice here?

Reply:
Yeah, I agree, we can scope this to currently reported issues only. I think no need to address this.